src/pages/ReportDetailsPage.tsx

```diff
@@ -117,24 +118,33 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         }
 
         // The Members page is only shown when:
+        // - The report is a thread in a chat report
         // - The report is not a user created room with participants to show i.e. DM, Group Chat, etc
         // - The report is a user created room and the room and the current user is a workspace member i.e. non-workspace members should not see this option.
-        if ((!isUserCreatedPolicyRoom && participants.length) || (isUserCreatedPolicyRoom && isPolicyMember)) {
+        if (
+            ((isDefaultRoom && isChatThread && isPolicyMember) ||
+                (!isUserCreatedPolicyRoom && participants.length) ||
+                (isUserCreatedPolicyRoom && (isPolicyMember || (isChatThread && !ReportUtils.isPublicRoom(report))))) &&
+            !ReportUtils.isConciergeChatReport(report)
```

Comment:
NAB: these conditions are super confusing and I'd be surprised if somebody doesn't break these eventually. No idea how to simplify things, but something we should consider in case we have to revisit this

Reply:
Yeah I agree, I was thinking of ways to simplify them but that would involve touching a bunch of other flows, so I elected to keep them the same for now so we could get them out sooner.